sregistry/client/share.py

```diff
@@ -15,21 +15,17 @@ def main(args, parser, subparser):
        control to a contact, usually an email.
     '''
     from sregistry.main import get_client
-    images = args.image
+    image = args.image
 
-    if not isinstance(images,list):
-        images = [images]
-
-    for image in images:
-        print(image)
+    print(image)
```

Comment:
This is probably a debug statement that should have been removed (oups).

Reply:
Fixed